src/commands/timeout.ts

```diff
@@ -0,0 +1,160 @@
+//Template: https://github.com/Costpap/CostBot/blob/main/src/commands/kick.ts
+
+//imports
+import { ChatInputCommandInteraction, PermissionFlagsBits, SlashCommandBuilder } from 'discord.js';
+import { notifyUser } from '../utils/misc';
+import ms from 'ms';
+
+export default {
+    data: new SlashCommandBuilder()
+        .setName('timeout')
+        .setDescription('Manages the timeout of a selected user.')
```

Comment:
Please change the string here to be consistent with the one initially written in the gist.
```suggestion
        .setDescription('Manages user timeouts.')
```